src/client/src/rt-system/ServiceStub.ts

```diff
@@ -1,11 +1,16 @@
 import { Error, ISubscription } from 'autobahn'
+import logdown from 'logdown'
 import { NextObserver, Observable } from 'rxjs'
 import { filter, mergeMap, switchMap } from 'rxjs/operators'
 
 import './AutoBahnTypeExtensions'
 import { ConnectionEvent, ConnectionEventType, ConnectionOpenEvent } from './connectionStream'
 
 const LOG_NAME = 'Connection:'
+const logger = logdown(`app:${LOG_NAME} Info `, { prefixColor: 'Olive' })
+const subLogger = logdown(`app:${LOG_NAME} Subscribing `, { prefixColor: 'MediumBlue' })
+const debugLogger = logdown(`app:${LOG_NAME}`, { prefixColor: 'ForestGreen' })
+const errorLogger = logdown(`app:${LOG_NAME} Error `, { prefixColor: 'DarkRed' })
```

Comment:
Does log down provide no other mechanism to achieve the color variation you're attempting to achieve?